### ISSUE TITLE
Sprint 2 planning

### ISSUE THREAD

**Issue by scharffc** [paceuniversity/CS3892017teamZACS#40]
It is missing which will penalize you for sprint 2.

**Comment by arizelee** [comment]
@scharffc Hello Professor. We added all of the issues for Sprint 2. Just to be clear, it is because there is no link to these issues. Correct?

**Comment by scharffc** [comment]
No. Please look at sprint 1 and read the documentation concerning sprint
planning. Your team has 5 members. I expect you to be on the top of the
process related deliverables.
